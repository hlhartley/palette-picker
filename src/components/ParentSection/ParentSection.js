import React from 'react';
import PaletteSection from '../../components/PaletteSection/PaletteSection';
import ProjectSection from '../../components/ProjectSection/ProjectSection';

export const ParentSection = () => {
    return (
        <div className="ParentSection">
            <div className="section-headers">
                <h2>PALETTE 1</h2>
                <h2>PROJECT 1</h2>
            </div>
            <div className="section-container">
                <PaletteSection/>
                <ProjectSection/>
            </div>
        </div>
    )
}

export default ParentSection;