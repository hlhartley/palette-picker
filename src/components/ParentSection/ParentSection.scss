.ParentSection {
  border-bottom: 1px solid black;
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  height: 90vh;
}