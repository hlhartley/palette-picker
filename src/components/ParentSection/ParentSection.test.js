import React from 'react';
import { shallow } from 'enzyme';
import { ParentSection } from './ParentSection';

describe('ParentSection', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <ParentSection/>
      )
    });
  it('should properly render the component elements', () => {
    expect(wrapper).toMatchSnapshot();
  });
});