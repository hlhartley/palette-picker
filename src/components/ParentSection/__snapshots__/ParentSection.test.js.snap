// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParentSection should properly render the component elements 1`] = `
<div
  className="ParentSection"
>
  <PaletteSection />
  <ProjectSection />
</div>
`;
